web/src/main/java/org/openmrs/web/filter/initialization/InitializationFilter.java

```diff
@@ -1133,6 +1153,12 @@ private void importTestDataSet(InputStream in, String connectionUrl, String conn
 		}
 	}
 	
+	private boolean checkDatabase(String database) {
+		if (wizardModel.databaseConnection.contains(database))
```

Comment:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-CodeStyle

Reply:
This is a mistake on my part. I'll correct it.